src/vtcomposite.cpp

```diff
@@ -148,10 +148,9 @@ struct CompositeWorker : Nan::AsyncWorker
                     vtzero::data_view tile_view{};
                     if (deflate::is_compressed(tile_obj->data.data(), tile_obj->data.size()))
                     {
-                        buffer_cache.emplace_back();
-                        std::string& buf = buffer_cache.back();
-                        decompressor(buf, tile_obj->data.data(), tile_obj->data.size());
-                        tile_view = vtzero::data_view{buf};
+                        buffer_cache.push_back(std::make_unique<std::vector<char>>());
```

Comment:
@artemp - what is the benefit of using `std::make_unique<std::vector<char>>()` over `std::string`?

Reply:
Copy&paste from `master`. I don't recall how we ended up with `std::make_unique<std::vector<char>>()` , perhaps it's just a residue from attempting to fix weird memory management issues .. hmm I'll need to check

Reply:
@springmeyer - changed to `std::vector<std::vector<char>>`. Note, `vector` is a better choice for storing binary data as it communicates intent clearer. Besides, pre c++11 (c++03) didn't guarantee std::string memory contiguity. Also, `string` will need to allocate one extra `\0`. None of these are problematic in practice but readability is important, too.

Reply:
c15df0941c0693d35090e511bd6e9c4534366b38